lib/chat_api_web/views/csv_helpers.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ChatApiWeb.CSVHelpers do
```

Comment:
could we add some tests for this module as well? (sorry I missed this in the first review!)